app/backplane/radio_module/src/c_udp_listener_tenant.cpp

```diff
@@ -15,7 +15,9 @@ void CUdpListenerTenant::PostStartup() {
 
 void CUdpListenerTenant::Run() {
     NTypes::RadioBroadcastData radioBroadcastData{0};
-    // Note len argument is the size of the data buffer, not how much data to receive! rcvResult will contain the actual amount of data received or -1 on error
+
+    // Note len argument is the size of the data buffer, not how much data to receive!
+    // rcvResult will contain the actual amount of data received or -1 on error
     const int rcvResult = udp.ReceiveAsynchronous(&radioBroadcastData.data, sizeof(radioBroadcastData.data));
```

Comment:
is the circumstance for `rcvResult` not equalling `len` argument:
- packet that comes in is of different size
- times out before enough bytes come in to get `len` bytes. 
I assume its number 1 bc udp but it would be nice to include that here in the docstring since ur talking about it anyways

Reply:
Actually number 1. You can still receive less bytes than the size of the buffer which is fine. This is by design because "we don't know" (we do, but don't want to hardcode this into radio module) the actual size of the telemetry. We just know it'll be less than the max size of a lora packet.